applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+	bool "Handle DESKTOP_DVFS_STATE_$(dvfs_state)"
+	default y
+	depends on $(dvfs_dependencies)
+
+if DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+
+if $(dvfs_timeout) != 0
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int "timeout"
```

Could you merge these two Kconfig option variants into one by using some kind of if syntax? Would this work?

```suggestion
	int "timeout" if $(dvfs_timeout) != 0
```